app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeries.java

```diff
@@ -32,6 +32,8 @@ public TaggedEventSeries createFromParcel(Parcel source) {
     private int mLogoResId;
     private int mSpecialEventTheme;
     private int mDrawerId;
+    private int mGreetingsResId;
+    private int mGreetingsTitleResId;
```

Comment:
Oh, right. Maybe we don't need separate greeting for each event series. We can have a common one for all of them.